### DISCUSSION TITLE
[Feed problem] https://www.howtogeek.com/feed/ AND https://www.makeuseof.com/feed/

### DISCUSSION THREAD

**Discussion by zlatco** [FreshRSS/FreshRSS#7125] [Q&A] [answered]
### Feed URL

https://www.howtogeek.com/feed/ AND https://www.makeuseof.com/feed/

### Describe the problem

Cant sign up for feeds
I can if using Newsblur or Inoreader.

### Log messages

cURL error 28: Connection timed out after 20002 milliseconds [https://www.howtogeek.com/feed/]

cURL error 28: Connection timed out after 20002 milliseconds [https://www.makeuseof.com/feed/]

### W3C Validator

Yes its valid
https://validator.w3.org/feed/check.cgi?url=https%3A%2F%2Fwww.howtogeek.com%2Ffeed%2F

### FreshRSS version

1.24.3

### Environment information

All installation checks is green.
Database version: [?]
- PHP version: [PHP version (8.2.26)]
- Installation type: [Softalicious] on Hostkoala
- Web server type: [?]
- Device: [PC]
- OS: [Win11]
- Browser: [Brave 1.73.101]

### Curl commands

Making a Feedburner works

### User agents

Yes.
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36

Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:133.0) Gecko/20100101 Firefox/133.0

Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:86.0)

**Comment by patman62** [comment, marked as answer]
Both feeds work for me - looks like they could be blocking your IP.

**Comment by zlatco** [comment]
It looks like you could be right, they do not like my server IP.
 THX
